### PR TITLE
chore(website): [playground] option to disable scroll and display tokens

### LINKED ISSUE #6769: Website: [Playground] ast viewer should display comments and tokens
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

while debugging [issue #6768](https://github.com/typescript-eslint/typescript-eslint/pull/6768) and trying to reproduce it in playground i stumbled upon issue with playground not displaying comments correctly

note:
- tokens and comments are currently hidden
  https://github.com/typescript-eslint/typescript-eslint/blob/bc31078cf86d69eee881e4a7daeffa347c1d82a7/packages/website/src/components/ast/serializer/serializerESTree.ts#L6
- auto-selecting nodes from editor should not trigger for tokens and comments

### Affected URL(s)

https://typescript-eslint.io/play/

### PR DESCRIPTION
**Pull request by armano2**
## PR Checklist

- [x] Addresses an existing open issue: fixes #6769
- [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [x] Steps in [Contributing](https://typescript-eslint.io/contributing) were taken

## Overview

- always display comments in ast
- allow to display tokens by enabling flag
- allow to disable scrolling -> this is useful for mobile devices

### CODE REVIEW

File: packages/website/src/components/Playground.tsx [hunk 1]
```diff
@@ -37,6 +37,9 @@ function Playground(): JSX.Element {
   const [visualEslintRc, setVisualEslintRc] = useState(false);
   const [visualTSConfig, setVisualTSConfig] = useState(false);
 
+  const [enableScrolling, setEnableScrolling] = useState<boolean>(true);
+  const [showTokens, setShowTokens] = useState<boolean>(false);
```
Comment (armano2) [inline packages/website/src/components/Playground.tsx:41]: do we want to save this in url / local storage?

Comment (JoshuaKGoldberg) [reply packages/website/src/components/Playground.tsx:41]: I think so, yes please!

File: packages/website/src/components/OptionsSelector.tsx [hunk 2]
```diff
@@ -36,26 +35,16 @@ function OptionsSelectorContent({
...
     window
       .open(
-        `https://github.com/typescript-eslint/typescript-eslint/issues/new?${createMarkdownParams(
-          state,
-        )}`,
+        `https://github.com/typescript-eslint/typescript-eslint/issues/new?${params}`,
         '_blank',
       )
       ?.focus();
-  }, [state, isLoading]);
```
Comment (JoshuaKGoldberg) [inline packages/website/src/components/OptionsSelector.tsx:58]: Oh, ha, yes, +1 on the `isLoading` check removal

File: packages/website/src/components/ast/utils.ts [hunk 3]
```diff
@@ -227,8 +228,12 @@ export function filterProperties(
   }
 
   switch (type) {
-    case 'esNode':
-      return key !== 'tokens' && key !== 'comments';
+    case 'esNode': {
+      if (key === 'tokens') {
+        return showTokens === true;
+      }
+      return true;
```
Comment (JoshuaKGoldberg) [inline packages/website/src/components/ast/utils.ts:235]: [Nitpick] Could simplify this a teeny bit, if you're interested?

```suggestion
      return key !== 'tokens' || !!showTokens;
```

**Review by JoshuaKGoldberg — APPROVED** [review]
💯 this is great, thanks! I've always found tokens to be irksome to work with without playground viewing support.

**Pull request merged**